modules/microshift-install-rhel-image-mode-run-container.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-run-container_{context}"]
+= Running the {microshift-short} bootc container
+
+If the host is already configured to have a `rhel` volume group with free space, this configuration is inherited by the container. It can be used by the {microshift-short} LVMS Container Storage Interface (CSI) to allocate storage.
```

Comment:
I do not think this section belongs here. We've already described the CSI options in the previous one.